astronomer/providers/google/cloud/hooks/dataproc.py

```diff
@@ -43,9 +54,18 @@ def get_job_client(
         :param location: (To be deprecated). The Cloud Dataproc region in which to handle the request.
         """
         client_options, region = self._get_client_options_and_region(region=region, location=location)
-        return JobControllerAsyncClient(
-            credentials=self._get_credentials(), client_info=CLIENT_INFO, client_options=client_options
-        )
+        try:
+            return JobControllerAsyncClient(
+                credentials=self._get_credentials(),
+                client_info=CLIENT_INFO,
+                client_options=client_options,
+            )
+        except AttributeError:  # pragma: no cover
+            return JobControllerAsyncClient(
+                credentials=self.get_credentials(),  # type: ignore[attr-defined]
+                client_info=CLIENT_INFO,
+                client_options=client_options,
+            )
```

Comment:
same comment as https://github.com/astronomer/astronomer-providers/pull/680/files#r983683544

Reply:
fixed it